logstash_async/handler.py

```diff
@@ -34,8 +33,6 @@ class AsynchronousLogstashHandler(Handler):
                       the database. (Given in seconds. Default is None, and disables this feature)
```

Comment:
`database_path` and `event_ttl` are relevant only for asynchronous processing.
I'd suggest to remove them here and move those two to `AsynchronousLogstashHandler`.

Reply:
Done